elasticapm/contrib/flask/__init__.py

```diff
@@ -133,7 +133,7 @@ def init_app(self, app, **defaults):
             self.client = make_client(self.client_cls, app, **defaults)
 
         # 0 is a valid log level (NOTSET), so we need to check explicitly for it
-        if self.logging or self.logging is 0:  # noqa F632
+        if self.logging or self.logging == 0:
```

Comment:
could we instead compare with `logging.NOTSET`? That would be more explicit and also suppresses the `SyntaxWarning`

```suggestion
        if self.logging or self.logging is logging.NOTSET:
```

(`logging` needs to be added to imports for this to work)

Reply:
Addressed.